src/app/components/nav-footer/nav-footer-styles.scss

```diff
@@ -0,0 +1,81 @@
+@import '~styles/variables.scss';
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 20px;
+  width: 100%;
+}
+
+.controls {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 180px;
+}
+
+.back {
+  position: relative;
+  color: $color-white;
+  font-family: 'PierSans';
```

Comment:
was this needed? I thought the whole page uses PierSans

Reply:
you're right I removed it